dist/index.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
```

Comment:
remove this file from the git and place `./dist` in the `.gitignore`

Reply:
Done